config/database.yml

```diff
@@ -2,9 +2,9 @@ default: &default
   adapter: mysql2
   encoding: utf8mb4
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  username: root
```

Comment:
This is my mistake. I shouldn't have uploaded the development DB credentials here. I changed them locally and they are imported through ENV variables.